calabash/Classes/FranklyServer/Operations/LPClearTextOperation.m

```diff
@@ -0,0 +1,29 @@
+#if ! __has_feature(objc_arc)
+#warning This file must be compiled with ARC. Use -fobjc-arc flag (or convert project to ARC).
+#endif
+
+#import "LPClearTextOperation.h"
+
+@implementation LPClearTextOperation
+
+- (id)performWithTarget:(id)target error:(NSError * __autoreleasing *) error {
+  if ([target respondsToSelector:@selector(setText:)]) {
+    [target performSelector:@selector(setText:) withObject:@""];
+    [self fireNotifications:target];
+    return target;
+  }
+
+  return nil;
+}
+
+- (void)fireNotifications:(id)target {
+  if ([target respondsToSelector:@selector(delegate)]) {
+    id delegate = [target performSelector:@selector(delegate)];
+
+    if ([delegate respondsToSelector:@selector(textFieldShouldClear:)]) {
+      [delegate performSelector:@selector(textFieldShouldClear:)];
+    }
+  }
+}
+
```

Comment:
We also need to call: `- textField:shouldChangeCharactersInRange:replacementString:`

```
@param string
The replacement string for the specified range. During typing, this parameter normally contains only the single new character that was typed, but it may contain more characters if the user is pasting text. When the user deletes one or more characters, the replacement string is empty.
```

This will work only for UITextField views.  A universal solution should work with:

* UITextField
* UITextView
* UISearchBar
* id<UIKeyInput>